src/MSALWrapper/TaskExecutor.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.MSALWrapper.AuthFlows
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Logging;
+
+    /// <summary>
+    /// The task executor.
+    /// </summary>
+    internal class TaskExecutor

```

Comment:
Can this be a static class? (i.e. you should not be able to create an instance of it)

Reply:
Yes, we should be able to.